sdk/communication/AzureCommunicationChat/Source/Models/ChatMessage.swift

```diff
@@ -52,6 +52,8 @@ public struct ChatMessage: Codable {
     public let deletedOn: Iso8601Date?
     /// The last timestamp (if applicable) when the message was edited. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`.
     public let editedOn: Iso8601Date?
+    /// Message metadata.
```

Comment:
Can we add more context to comment. What exactly is expected in the metadata? The `[String: String?]` doesn't exactly give developers much context